hedera-node/src/main/java/com/hedera/services/txns/contract/process/CreateEvmTxProcessor.java

```diff
@@ -57,10 +58,11 @@ public TransactionProcessingResult execute(
 			final long providedGasLimit,
 			final long value,
 			final Bytes code,
-			final Instant consensusTime) {
+			final Instant consensusTime,
+			final Optional<Long> expiry) {
```

Comment:
We can remove the `Optional` property here as we are always going to provide it as parameter in `CreateEvmTxProcessor`